requirements/requirements.txt

```diff
@@ -10,6 +10,6 @@ trading_calendars>=1.11
 click==8.0.1
 PyYAML>=5, <6
 ipython>=7
-alpaca-trade-api>=1.2.2
+alpaca-trade-api==1.5.0
```

Comment:
Why are we using v1.5.0 instead of 2.0.0?

Reply:
The answer to that is in the top comment on the PR there :see_no_evil: